src/BitStringAddresses/fockaddress.jl

```diff
@@ -129,76 +130,108 @@ Iterate over all occupied modes in an address. Iterates [`BoseFSIndex`](@ref) fo
 julia> b = BoseFS((1,5,0,4));
 
 julia> foreach(println, occupied_modes(b))
-[1, 1, 0]
-[5, 2, 2]
-[4, 4, 9]
+BoseFSIndex(occnum=1, mode=1, offset=0)
+BoseFSIndex(occnum=5, mode=2, offset=2)
+BoseFSIndex(occnum=4, mode=4, offset=9)
 ```
 
 ```jldoctest
 julia> f = FermiFS((1,1,0,1,0,0,1));
 
 julia> foreach(println, occupied_modes(f))
-1
-2
-4
-7
+FermiFSIndex(occnum=1, mode=1)
+FermiFSIndex(occnum=1, mode=2)
+FermiFSIndex(occnum=1, mode=4)
+FermiFSIndex(occnum=1, mode=7)
 ```
 """
 occupied_modes
 
 """
-    move_particle(add::SingleComponentFockAddress, i, j) -> nadd, α
+    excitation(a::SingleComponentFockAddress, creations::NTuple{N}, destructions::NTuple{N})
 
-Move particle from mode `i` to mode `j`. Returns the new Fock state address `nadd` and
-amplitude `α`. Equivalent to
-```math
-a^{\\dagger}_i a_j |\\mathrm{add}\\rangle \\to α|\\mathrm{nadd}\\rangle
-```
+Generate an excitation on address `a` by applying `creations` and `destructions`, which are
+tuples of the appropriate address indices (i.e. integers for fermions and `BoseFSIndex` for
+bosons).
```

Comment:
```suggestion
Generate a number-conserving excitation on address `a` by applying `creations` and `destructions`, 
which are tuples of the appropriate address indices (i.e. [`FermiFSIndex`](@ref) for fermions and
[`BoseFSIndex`](@ref) for bosons).
```math
a^{\\dagger}_{c_1} a^{\\dagger}_{c_2} \\ldots a_{d_1} a_{d_2} \\ldots |\\mathrm{a}\\rangle \\to α|\\mathrm{nadd}\\rangle
```
```